_technical/structured-logging.md

```diff
@@ -1,44 +1,35 @@
 ---
```

Comment:
As per @darren-reddick comment, what does the rest of the @ContainerSolutions/cre thinks about this technical pattern? Should we try to refactor it or just get rid of it for the time being?

Reply:
Structured logging is definitely legitimate thing, and refers to the tokenisation of logs. So in terms of standard fields, and also in terms of formatting, whether that be JSON or comma separated with a fixed field order, etc...